cmd/podman/pod_stats.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"encoding/json"
+	tm "github.com/buger/goterm"
+	"github.com/containers/libpod/cmd/podman/formats"
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/containers/libpod/libpod"
+	"github.com/pkg/errors"
+	"github.com/ulule/deepcopier"
+	"github.com/urfave/cli"
+)
+
+var (
+	podStatsFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "show stats for all pods.  Only running pods are shown by default.",
+		},
+		cli.BoolFlag{
+			Name:  "no-stream",
+			Usage: "disable streaming stats and only pull the first result, default setting is false",
+		},
+		cli.BoolFlag{
+			Name:  "no-reset",
+			Usage: "disable resetting the screen between intervals",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "pretty-print container statistics to JSON or using a Go template",
+		}, LatestFlag,
```

Comment:
This should also be LatestPodFlag if latest is being included

Reply:
fixed